app/views/cars/_slotbook.html.erb

```diff
@@ -1,9 +1,10 @@
 <h1> Book a car </h1>
+<%#what is this file for if everything is commented here %>
```

Comment:
This file is deleted in the main branch because we're using requests now instead of slotbook.